polaris-service-quarkus/build.gradle.kts

```diff
@@ -220,6 +220,15 @@ tasks.withType(Test::class.java).configureEach {
   addSparkJvmOptions()
 }
 
+tasks.named<Test>("test").configure {
+  if (System.getenv("AWS_REGION") == null) {
+    environment("AWS_REGION", "us-west-2")
+  }
+  jvmArgs("--add-exports", "java.base/sun.nio.ch=ALL-UNNAMED")
+  useJUnitPlatform()
+  maxParallelForks = 4
```

Comment:
We must be careful with many forks running Quarkus tests, you could experience OOM issues. But let's leave this for now if tests are passing.